rpm/patches/01-rlimit_nofile-rhel7.patch

```diff
@@ -0,0 +1,15 @@
+diff --git i/contrib/init/systemd/docker.service w/contrib/init/systemd/docker.service
+index d8c7867057..b73ecf7363 100644
+--- i/contrib/init/systemd/docker.service
++++ w/contrib/init/systemd/docker.service
+@@ -30,6 +30,10 @@ StartLimitInterval=60s
+ # in the kernel. We recommend using cgroups to do container-local accounting.
+ LimitNPROC=infinity
+ LimitCORE=infinity
++# Older systemd versions default to a LimitNOFILE of 1024:1024, which is insufficient for many
++# applications including dockerd itself and will be inherited. Raise the hard limit, while
++# preserving the soft limit for select(2).
++LimitNOFILE=1024:524288
+ 
+ # Comment TasksMax if your systemd version does not support it.
+ # Only systemd 226 and above support this option.
```

Comment:
Unrelated, but now wondering what version of systemd RHEL/CentOS 7 are running, and if this comment is still relevant (ISTR it was always just a warning, so not even sure if relevant _at all_)

Reply:
219: https://centos.pkgs.org/7/centos-x86_64/systemd-sysv-219-78.el7.x86_64.rpm.html